.github/workflows/codeql-analysis.yml

```
@@ -2,6 +2,7 @@ name: "Code scanning - action"
 
 on:
   push:
+  pull_request:
     paths-ignore:
       - '**.md'
```

```suggestion
  push:
      paths-ignore:
      - '**.md'
  pull_request:
    paths-ignore:
      - '**.md'
```
`paths-ignore` only applies to `pull_request` here, should we have it apply to both `push` and `pull_request`?

---

That's a good question. It is currently required to have the base commit run without filtering so that when you open a PR we definitely have results for the current HEAD of the base branch to compare against.

One thing you could do is limit both push and pull_request to the branches that you'll make PRs against (typically just main).